### PR TITLE
Add distributed mode configuration

### PR DESCRIPTION
**Pull request by ikbalkaya**
This PR adds an alternative Docker-compose file for distributed mode configuration and updates Readme to include instructions on distributed setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,23 +56,39 @@ To manually install the connector on a local installation of Confluent:
...
+   `docker-compose -f docker-compose-distributed.yml up -d` for distributed mode.
+   
+**Note**: You can view the logs using `docker-compose logs connector`
+
+**Note 2**: You must start your connectors using [Connect REST interface](https://docs.confluent.io/platform/current/connect/references/restapi.html) when using distributed mode.
 
-    **Note**: You can view the logs using `docker-compose logs connector`
 
-3. Once the containers have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
+4. Once the containers have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
```
Comment (KacperKluka) [inline README.md:85]: Why has the numbering changed and now there's `4` after `2`? :thinking:

File: docker-compose-distributed.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
+    networks:
+      kafka_distributed_network:
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      zk_id: '1'
+    # ports:
+    #  - '2181:2181'
```
Comment (KacperKluka) [inline docker-compose-distributed.yml:11]: Shouldn't we remove those commented lines? :thinking:

File: docker-compose-distributed.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+
+  schema-registry:
+    image: confluentinc/cp-schema-registry:5.4.0
+    networks:
+      kafka_distributed_network:
+        aliases:
+          - schema-registry
+    # ports:
+    # - '8081:8081'
```
Comment (KacperKluka) [inline docker-compose-distributed.yml:40]: Like above, shouldn't we remove them? :thinking:

File: config/docker-compose-worker-distributed.properties [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+status.storage.replication.factor=1
+
+
+# The config storage topic must have a single partition, and this cannot be changed via properties.
+# Offsets for all connectors and tasks are written quite frequently and therefore the offset topic
+# should be highly partitioned; by default it is created with 25 partitions, but adjust accordingly
+# with the number of connector tasks deployed to a distributed worker cluster. Kafka Connect records
+# the status less frequently, and so by default the topic is created with 5 partitions.
+#offset.storage.partitions=25
+#status.storage.partitions=5
```
Comment (KacperKluka) [inline config/docker-compose-worker-distributed.properties:52]: In this properties file there are a lot of commented lines and information about stuff that we are not using, I know that they might be helpful in the future but are we okay with committing them to the repository? :thinking:

**Review by KacperKluka — COMMENTED** [review]
Looks good but I have a few questions :wink:

File: README.md (hunk 1 above)
Comment (ikbalkaya) [reply README.md]: Oops, that must have happened automatically by entering. Fix is 37661743b5f394e4c239c269ccae938d0e3791e2

File: docker-compose-distributed.yml (hunk 2 above)
Comment (ikbalkaya) [reply docker-compose-distributed.yml:11]: These were on original configuration files so I did not remove them. Users arguably might want to open those ports to outside, so I'd say we can keep them commented here

File: docker-compose-distributed.yml (hunk 3 above)
Comment (ikbalkaya) [reply docker-compose-distributed.yml:40]: The same as above

File: config/docker-compose-worker-distributed.properties (hunk 4 above)
Comment (ikbalkaya) [reply config/docker-compose-worker-distributed.properties:52]: These comments were part of example configuration file. In my opinion it's good to keep those commented code on configuration files to help customers about what they can use. After all, these configuration files are just examples and will eventually be overriden (or not even used) by customers.

**Review by KacperKluka — APPROVED** [review]
LGTM :+1:

**Pull request merged**